Automattic/fresh-data
# Split out packages using lerna

This splits out fresh-data into two packages:
  @fresh-data/framework
  @fresh-data/react-provider

The purpose of this is to split out the react/redux code so the `framework` bundle can be smaller.

Even with including what is needed from `lodash` and `debug` in the bundle, the `framework` bundle is 7001 bytes minified.

To Test:
`npm install`
`npm run bootstrap`
`npm run build`
`npm run test`
Ensure all tests pass from both packages.
